bandwidth-file-spec.txt

```diff
@@ -278,6 +282,28 @@
 
       This Line was added in version 1.2.0 of this specification.
 
+    "scanner_country=" CountryCode NL
+
+      [Zero or one time.]
+
+      The country, as in political geolocation, where the generator is run.
+
+      This Line was added in version 1.3.0 of this specification.
+
+    "destinations_countries=" CountryCodeList NL
+
+      [Zero or one time.]
+
+      The country, as in political geolocation, or countries where the
+      destination Web server(s) are located.
+      The destination Web Servers serve the data that the generator retrieves
+      to measure the bandwidth.
+
+      The code "ZZ" is used here to denote that the country is unknown.
```

Comment:
Do we need to redefine ZZ here? We already did it above when defining CountryCode, right? Perhaps we can also give the CDN example above where CountryCode is defined, and remove this from here.

Reply:
Sounds good. I just didn't want to make the definition too long, nor to to use CDN abbreviation.
I've instead used "eg", which i found is used some specs, so seems to be ok.
See fixup.